### PR TITLE
add advice to delete data via recorder services

### PR DESCRIPTION
**Pull request by copykatze**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @copykatze, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20445).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_docs/backend/database.markdown [hunk 1]
```diff
@@ -127,6 +127,10 @@ group.all_switches              8018
 
 If you don't want to keep certain entities, you can delete them permanently:
 
+To do so, it is best to use the services provided by the recorder: https://www.home-assistant.io/integrations/recorder/#service-purge_entities
+
+Deleting data directly from the database might lead to a corrupted database, since the database isn't meant to be consumed like that:
```
Comment (frenck) [inline source/_docs/backend/database.markdown]: Let's remove the manual deletion part.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by homeassistant** [comment]
Hi @copykatze, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20445).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by copykatze — COMMENTED** [review]
removed the manual deletion part

File: source/_docs/backend/database.markdown [hunk 2]
```diff
@@ -125,20 +125,6 @@ group.all_switches              8018
...
-```bash
-sqlite> DELETE FROM states WHERE entity_id="sensor.cpu";
-```
-
-The `VACUUM` command cleans your database.
-
-```bash
-sqlite> VACUUM;
-```
+If you don't want to keep certain entities, you can delete them permanently by using the services provided by the recorder: https://www.home-assistant.io/integrations/recorder/#service-purge_entities
```
Comment (frenck) [inline source/_docs/backend/database.markdown]:
```suggestion
If you don't want to keep certain entities, you can delete them permanently by using the [services provided by the recorder](/integrations/recorder/#service-purge_entities).
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by homeassistant** [comment]
Hi @copykatze, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20445).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @copykatze 👍

**Pull request merged**